### ISSUE TITLE
Can upload pre-trained models for CIFAR10 and ImageNet?

### ISSUE THREAD

**Issue by finlay-liu** [oval-group/smooth-topk#1]
Hi, Leonard

Great job, what a nice paper!
Can you upload your pre-trained models for CIFAR10 and ImageNet?
It's hard for me to re-train all the models. Thanks.

Yuzhong.

**Comment by lberrada** [comment]
Hi Yuzhong, the models are now available for download [here](http://www.robots.ox.ac.uk/~lberrada/smooth-topk-data/). 

At the time of the experiments, the pre-processing mean and variance were unnecessarily divided by 255 for ImageNet. Therefore, if you are using the models trained on ImageNet, make sure to use the following statistics in the [data loading code](https://github.com/oval-group/smooth-topk/blob/master/src/data/main.py#L85):
```python
mean = [x / 255. for x in [0.485, 0.456, 0.406]]
std = [x / 255. for x in [0.229, 0.224, 0.225]]
```
Note that this does not impact the validity of the results, because the affine transformation on the input is cancelled out by the first batch normalization layer. I will reproduce the experiments without this issue when I have more resources available.

**Comment by finlay-liu** [comment]
thank you, Leonard!
